modules/tinymce/src/core/test/ts/browser/DragDropOverridesTest.ts

```diff
@@ -8,198 +8,288 @@ import { assert } from 'chai';
 import Editor from 'tinymce/core/api/Editor';
 
 describe('browser.tinymce.core.DragDropOverridesTest', () => {
-  const fired = Cell(false);
-  const hook = TinyHooks.bddSetup<Editor>({
-    indent: false,
-    menubar: false,
-    base_url: '/project/tinymce/js/tinymce',
-    height: 300,
-    width: 300,
-  }, [], true);
-
-  before(() => {
-    hook.editor().on('dragend', () => {
-      fired.set(true);
+
+  context('Tests without autoresize', () => {
+    const fired = Cell(false);
+    const hook = TinyHooks.bddSetup<Editor>({
+      indent: false,
+      menubar: false,
+      base_url: '/project/tinymce/js/tinymce',
+      height: 300,
+      width: 300,
+    }, [], true);
+
+    before(async () => {
+      hook.editor().on('dragend', () => {
+        fired.set(true);
+      });
+      await Waiter.pWait(100); // Wait a small amount of time to ensure the events have been bound
```

Comment:
Curious why is this needed?

Reply:
I don't know. It was there already 🤷‍♂️

Reply:
From memory it's because some tests here check the drag/drop blocking behaviour and that's bound in the next event loop after initial runs, however the mcagar ready callback runs when init is fired in most cases. See https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/core/main/ts/DragDropOverrides.ts#L408-L411

Reply:
This conversation should be resolved then ?

Reply:
Weird that github shows that as a new line. Thought it was part of this PR. Anyway I think this is resolved. Makes sense that it needs some time to finish binding the events.